capa/features/extractors/dnfile/insn.py

```diff
@@ -139,9 +139,9 @@ def extract_insn_property_features(fh: FunctionHandle, bh, ih: InsnHandle) -> It
             if property is None:
                 return
             if property[1] == DnPropertyAccessType.SET:
-                yield WriteProperty(str(property[0])), ih.address
+                yield Property(str(property[0]), "property/write"), ih.address
```

Comment:
I think this is more elegant. However, there may be some concerns with "mixing" feature syntax and internals here.
Any counter arguments to using a generic access type enum for the `Property` `access` field here instead and handle it later?

Also #1142 should solve this.

Reply:
I think this is a good argument for #1142.